esmvaltool/recipes/bock20jgr/recipe_bock20jgr_fig_8-10.yml

```diff
@@ -1,17 +1,16 @@
 # ESMValTool
```

Comment:
This is the only none yamllint compliant recipe of the PR. I leave it up to you to say if this would help readability of this recipe or not.